### ISSUE TITLE
refactor the footer of the docs with links to community. 

### ISSUE THREAD

**Issue by kmova** [openebs/openebs-docs#815]
This idea started with https://github.com/openebs/openebs-docs/pull/814#pullrequestreview-421654884

It will be good to actually have the footer display links to Slack, GitHub, Twitter in place of the current text. Something similar to: 

![image](https://user-images.githubusercontent.com/12488783/83403548-91cc0180-a426-11ea-89ad-b070e9411880.png)

**Comment by SandeepGamot** [comment]
I assume this isn't assigned to anyone.
I'll take this up if not.

**Comment by kmova** [comment]
@SandeepGamot, yes - this has not been completed. Please feel free to take this up.
